frontend/src/constants/index.ts

```diff
@@ -4,7 +4,7 @@ import { inSeconds } from '@/types/utils';
 
 const { publicRuntimeConfig, serverRuntimeConfig } = getConfig();
 
-export const UNDEFINED = 'UNDEFINED';
+export const UNDEFINED = ' UNDEFINED';
```

Comment:
Why did you add a space here?

```suggestion
export const UNDEFINED = 'UNDEFINED';
```